src/app/routing/resolve/decision.resolve.ts

```diff
@@ -11,9 +11,7 @@ export class DecisionResolve implements Resolve<any> {
     }
 
     resolve(route: ActivatedRouteSnapshot) {
-        console.log('resolve');
-        return this.decisionService.fetch(route.paramMap.get('case_id'))
-            .pipe(catchError(error => {
+        return this.decisionService.fetch(route.paramMap.get('case_id')).pipe(catchError(error => {
```

Comment:
try to put chained methods on new line -> this allows devs to put blocks when debugging.

Reply:
Yes I agree however unfortunately this also has a significant impact on code coverage :(